### DISCUSSION TITLE
[Q] How can I convert Excel to pdf?

### DISCUSSION THREAD

**Discussion by resobyte** [exceljs/exceljs#2394] [Q&A]
## 💬 Questions and Help

Hello,

**Can I convert the Excel I created to pdf?**

I tried,

```
 const xlsx = await workbook.xlsx.writeBuffer();

  attachments: [
            {
                filename: fileNameExcel+'.pdf',
                content: xlsx,
                contentType:
                    'application/pdf',
            },
        ]
```

**Finally**

<img width="548" alt="Screen Shot 2020-12-13 at 20 01 04" src="https://user-images.githubusercontent.com/17431441/102018377-ff65cc80-3d7d-11eb-9d22-d14694e664bb.png">

_Attachments is node-mailer package._

**Comment by shulamitco** [comment]
Did you do it finally?

**Comment by hohler** [comment]
`writeBuffer` will for sure create an xlsx file, not a .pdf file.

If you want to create a PDF, check out https://github.com/exceljs/exceljs/issues/1806
